server/src/version_provider.ts

```diff
@@ -6,65 +6,134 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import * as path from 'path';
+const MIN_TS_VERSION = '3.6';
```

Comment:
that's gonna be in the next PR ;) I want to separate the changes from the version upgrade.

Reply:
I think a language service will not work with ts 3.7 because the `typeArguments` property has been removed from the `TypeReference` interface in Typescipt 3.7. Two usages of `typeArguments` exist in @angular/.../typescript_symbols.ts file.

Reply:
Also, please see my comment here https://github.com/angular/vscode-ng-language-service/issues/437#issuecomment-557723449
I think 3.7.2 does not contain the fix

Reply:
merging since this PR doesn't deal with 3.7 yet.